schema/SCHEMA_QUDT-v2.1.ttl

```diff
@@ -1442,12 +1442,12 @@ qudt:Unit
   rdfs:subClassOf [
       a owl:Restriction ;
       owl:allValuesFrom qudt:Unit ;
-      owl:onProperty qudt:hasDenominatorPart ;
+      owl:onProperty qudt:qkdvDenominatorPart ;
```

Comment:
Do we need to append 'Part'?

Reply:
I agree. It should not point to a Unit (and it doesn't). It points to a dimension vector.

Reply:
Ah, I just noticed why you were talking about pointing to a Unit in the snippet above. Also noticed that you (or somebody) had already associated a different property with qudt:QuantityKind, namely qudt:numeratorDimensionVector (and one for denominator). I hadn't noticed that. Rather, I had just renamed the old qudt:hasDenominatorPart property, and neglected to even look at the schema definition for qudt:QuantityKind. That's my bad. 

So, since I need to go back and fix that, we might as well come to consensus on what we should call the property. Please voice your support for:
1. qudt:numeratorDimensionVector
2. qudt:numeratorQKDV
3. qudt:qkNumerator
4. Something else altogether.

Once we have consensus, I will fix the schema definition of qudt:QuantityKind, and update the small number of instances already populated.